ai2thor/server.py

```diff
@@ -85,15 +85,39 @@ def unique_rows(arr, return_index=False, return_inverse=False):
         return unique
 
 
-class Event(object):
+class MetadataWrapper(dict):
+    def __init__(self, raw_metadata: dict):
+        super().__init__(raw_metadata)
+        self._raw_metadata = raw_metadata
+
+    def __getitem__(self, x):
+        # alias deprecated functionality
+        if (
+            x == "reachablePositions"
+            and self._raw_metadata["lastAction"] == "GetReachablePositions"
```

Comment:
Doesn't GetSceneBounds also depend on reachablePositions?

Reply:
No. It looks like it just encapsulates the bounds of all Renderer's (i.e., object meshes that exist in the scene):
https://github.com/allenai/ai2thor/blob/main/unity/Assets/Scripts/AgentManager.cs#L370

![image](https://user-images.githubusercontent.com/28768645/113445157-318d4f80-93aa-11eb-8b31-c123c6d4154c.png)

Reply:
Fixed in https://github.com/allenai/ai2thor/pull/628/commits/184c7e8ddfa82a17f84cc10d260098df65b94544